README.md

```diff
@@ -14,13 +14,30 @@ The playbook build-cluster-ec2.yml launches three instances across two availabil
 You'll need the following in order to use these playbooks.
 
 * Access Key and Secret for an AWS Account with permissions to admin EC2.
-* Ansible installed on a controller host. The faster the controller host's connection to the choosen EC2 region, the faster nodes will launch. Ssh access to a small to medium instance in the same AWS region as your cluster works well. See Ansible Setup below for further details.
+* Ansible installed on a controller host (See **Ansible Setup** below for further details). The faster the controller host's connection to the choosen EC2 region, the faster nodes will launch. Ssh access to a small to medium instance in the same AWS region as your cluster works well.
 * An AWS Key Pair (PEM file) downloaded to the Ansible controller host.
 * A copy of the ConductR deb installation package on the Ansible controller host.
 * A copy of this GitHub repo on the Ansible controller host.
 * Ability to accept Oracle's Java License.
 
-## Setup
+## Ansible Setup
+
+```bash
+sudo apt-get install python-setuptools autoconf g++ python2.7-dev
+sudo easy_install pip
+sudo pip install ansible paramiko PyYAML Jinja2 httplib2 boto
```

Comment:
`pip` install `ansible` version `2.3.0` which is very up to date. So I think we can simplify instruction thanks to this. What do you think ?